filters/redirect/redirect.go

```diff
@@ -43,7 +44,21 @@ func (f *Redirect) Request(ctx skipper.FilterContext) {}
 
 func (f *Redirect) Response(ctx skipper.FilterContext) {
 	w := ctx.ResponseWriter()
-	w.Header().Set("Location", f.location)
+
+	u, err := url.Parse(f.location)
+
+	if err != nil {
+		w.WriteHeader(501)
```

Comment:
uhh, yeah... this is left open and i just forgot to ask it in the end. shall we return 50x errors, or only 404 in these cases? some people don't return 50x to avoid giving tips for attackers. what shall be our approach?